cpp/binary/viewer/CMakeLists.txt

```diff
@@ -101,4 +101,4 @@ install(TARGETS viewer
     RUNTIME DESTINATION "${INSTALL_VIEWERDIR}"
     BUNDLE DESTINATION "${INSTALL_VIEWERDIR}"
     LIBRARY DESTINATION "${INSTALL_VIEWERDIR}"
-)
\ No newline at end of file
+)
```

Comment:
Please do not make this change that is only unnecessary formatting.

Reply:
ditto